bundle-size.jsonc

```
@@ -6,6 +6,7 @@
   "schemaVersion": 1,
   "apps": {
     "@cowprotocol/cow-fi": 6130633, // 5.85 MiB
+    "@cowprotocol/cowswap": 12985009, // 12.38 MiB
```

Biome (2.5.3)

[error] The key `@cowprotocol/cowswap` was already declared.

(lint/suspicious/noDuplicateObjectKeys)